src/test/java/com/SAS/acceptanceTests/LeagueManagementAT.java

```diff
@@ -1,81 +1,207 @@
+            System.out.println("The referee added successfully");
+            System.out.println("System saved the data");
+        }
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.5.a test - success
+    @Test
+    void successfulDefinePointsPolicyAndRankPolicyForTheLeague() {
+        long startTime = System.nanoTime();
+        successfulDefineSeasonToLeague();
+        //9.5
+        System.out.println("Choose points policy");
+        league.addPointsPolicy(season, pPolicy);
+        System.out.println("You have chose: " + pPolicy.toString());
+        System.out.println("System saved the data");
+        Assertions.assertTrue(league.getPointsPolicy(season).getName().equals(pPolicy.getName()));
+        //9.5
+        System.out.println("Choose league rank policy");
+        league.addRankPolicy(season, rPolicy);
+        System.out.println("You have chose: " + rPolicy.toString());
+        System.out.println("System saved the data");
+        Assertions.assertTrue(league.getRankPolicy(season).getName().equals(rPolicy.getName()));
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.5.a test - success
+    @Test
+    void successfulDefineGamesPolicyForTheLeague() {
+        //9.6
+        long startTime = System.nanoTime();
+        successfulDefineSeasonToLeague();
+        System.out.println("Choose games policy");
+        league.addGamePolicy(season, gPolicy);
+        System.out.println("You have chose: " + gPolicy.toString());
+        System.out.println("System saved the data");
+        Assertions.assertTrue(league.getGamesPolicy(season).getName().equals(gPolicy.getName()));
+        System.out.println("League initialized successfully");
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.1.b test - failure
+    @Test
+    void failingLeagueCreation() {
+        //9.1
+        long startTime = System.nanoTime();
+        System.out.println("You have selected to edit association settings");
+        Assertions.assertFalse(false);
+        System.out.println("You are not authorized to enter this page");
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.2.b test - failure
+    @Test
+    void failingDefineSeasonToLeague() {
+        //9.2
+        long startTime = System.nanoTime();
+        System.out.println("You have selected to edit association settings");
+        System.out.println("There is the association page");
+        Assertions.assertFalse(CRUD.isLeagueExist(new League("Liga Leumit")));
```

Comment:
I think you need to add the function of "addSeasonToLeague" in the LeagueManagement controller and than it would return false.